UnityProject/Assets/Scripts/Health/Sickness/Sickness.cs

```diff
@@ -33,7 +34,20 @@ public class Sickness : MonoBehaviour
 		private int currentTicksSinceLastProgression = 0;
 
 		private int currentStage = 1;
-		public int CurrentStage => currentStage;
+		public int CurrentStage
+		{
+			get { return currentStage; }
+			set
+			{
+				if (value > NumberOfStages)
+				{
+					Logger.LogWarning("[Sickness] - Attempted setting stage for sickness that was bigger than the identified stages.");
```

Comment:
Warnings are way too easy to ignore, make it an error instead.

Reply:
Done